lib/models/entitlement_info_wrapper.g.dart

```diff
@@ -49,7 +49,7 @@ Map<String, dynamic> _$$_EntitlementInfoToJson(_$_EntitlementInfo instance) =>
 const _$OwnershipTypeEnumMap = {
   OwnershipType.purchased: 'PURCHASED',
   OwnershipType.familyShared: 'FAMILY_SHARED',
-  OwnershipType.unknown: 'unknown',
+  OwnershipType.unknown: 'UNKNOWN',
```

Comment:
This has changed. Is it ok? Looks like the new value is more accurate anyways

Reply:
Yeah I actually had to change this here: https://github.com/RevenueCat/purchases-flutter/pull/726/files#r1228355050

This is the actual value that gets sent from PHC so as you said, it's more correct. And I don't think this will cause any breaking changes at all (unless someone was deliberately trying to create an entitlement info with a `unknown` ownership type.